janusgraph-cql/pom.xml

```diff
@@ -0,0 +1,342 @@
+                        <goals>
+                            <goal>test</goal>
+                        </goals>
+                        <phase>test</phase>
+                        <configuration>
+                            <excludedGroups>${test.excluded.groups},org.janusgraph.testcategory.OrderedKeyStoreTests,org.janusgraph.testcategory.SerialTests,org.janusgraph.testcategory.CassandraSSLTests</excludedGroups>
+                            <groups />
+                            <reportNameSuffix>${test.murmur}</reportNameSuffix>
+                            <systemProperties>
+                                <property>
+                                    <name>test.cassandra.confdir</name>
+                                    <value>${project.build.directory}/cassandra/${test.murmur}/conf</value>
+                                </property>
+                                <property>
+                                    <name>test.cassandra.datadir</name>
+                                    <value>${project.build.directory}/cassandra/${test.murmur}/data</value>
+                                </property>
+                            </systemProperties>
+                            <skip>${test.skip.murmur}</skip>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>${test.murmur}-serial-test</id>
+                        <goals>
+                            <goal>test</goal>
+                        </goals>
+                        <phase>test</phase>
+                        <configuration>
+                            <excludedGroups />
+                            <groups>org.janusgraph.testcategory.SerialTests</groups>
+                            <parallel>none</parallel>
+                            <perCoreThreadCount>false</perCoreThreadCount>
+                            <threadCount>1</threadCount>
+                            <runOrder>alphabetical</runOrder>
+                            <reportNameSuffix>${test.murmur}-serial</reportNameSuffix>
+                            <systemProperties>
+                                <property>
+                                    <name>test.cassandra.confdir</name>
+                                    <value>${project.build.directory}/cassandra/${test.murmur}/conf</value>
+                                </property>
+                                <property>
+                                    <name>test.cassandra.datadir</name>
+                                    <value>${project.build.directory}/cassandra/${test.murmur}/data</value>
+                                </property>
+                            </systemProperties>
+                            <skip>${test.skip.murmur-serial}</skip>
+                        </configuration>
+                    </execution>
+                    <execution>
+                        <id>${test.murmur-ssl}-test</id>
```

Comment:
Regarding the serial and SSL tests, it appears to me that they both run by default in janusgraph-cassandra unless the appropriate skip property is defined.

I believe the janusgraph-cql module has the same behaviour but happy to change if this isn't correct.